tests/transport_test.go

```diff
@@ -123,14 +119,14 @@ var _ = Describe("Transport Tests", func() {
 
 	httpNoAuthEp := fmt.Sprintf("http://%s:%d", utils.FileHostName+"."+utils.FileHostNs, utils.HTTPNoAuthPort)
 	httpAuthEp := fmt.Sprintf("http://%s:%d", utils.FileHostName+"."+utils.FileHostNs, utils.HTTPAuthPort)
-	registryNoAuthEp := fmt.Sprintf("docker://%s", "docker.io")
+	registryNoAuthEp := fmt.Sprintf("docker://%s", utils.RegistryHostName+"."+utils.RegistryHostNs)
 	DescribeTable("Transport Test Table", it,
 		Entry("should connect to http endpoint without credentials", httpNoAuthEp, targetFile, "", "", controller.SourceHTTP, true),
 		Entry("should connect to http endpoint with credentials", httpAuthEp, targetFile, utils.AccessKeyValue, utils.SecretKeyValue, controller.SourceHTTP, true),
 		Entry("should not connect to http endpoint with invalid credentials", httpAuthEp, targetFile, "gopats", "bradyisthegoat", controller.SourceHTTP, false),
 		Entry("should connect to QCOW http endpoint without credentials", httpNoAuthEp, targetQCOWFile, "", "", controller.SourceHTTP, true),
 		Entry("should connect to QCOW http endpoint with credentials", httpAuthEp, targetQCOWFile, utils.AccessKeyValue, utils.SecretKeyValue, controller.SourceHTTP, true),
-		Entry("should connect to registry endpoint without credentials", registryNoAuthEp, "registry", "", "", controller.SourceRegistry, true),
-		//		Entry("should not connect to registry endpoint with invalid credentials", registryNoAuthEp, "registry", "gopats", "bradyisthegoat", controller.SourceRegistry, false),
+		Entry("should connect to registry endpoint without credentials", registryNoAuthEp, targetQCOWImage, "", "", controller.SourceRegistry, true),
+		Entry("should fail to import from registry endpoint when container image is invalid", registryNoAuthEp, targetRegistryInvalidImage, "", "", controller.SourceRegistry, false),
```

Comment:
I think the negtive case we want to conver is here: https://github.com/kubevirt/containerized-data-importer/pull/608/files#diff-9cbc435c7c64940bbba1090c17662234R309

should add a test about:
`should fail to import from registry endpoint when container image is not putting in  /disk directory` sth like this.